checkly/resource_dashboard.go

```diff
@@ -51,7 +51,25 @@ func resourceDashboard() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Default:     "",
-				Description: "A URL pointing to an image file.",
+				Description: "A URL pointing to an image file to use for the dashboard logo.",
+			},
+			"favicon": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "",
+				Description: "A URL pointing to an image file to use as browser favicon",
```

Comment:
The other descriptions end with a period but this doesn't, should we add it to keep it consistent?